src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceServiceTest.kt

```diff
@@ -88,8 +89,12 @@ class LicenceServiceTest {
   private val notifyService = mock<NotifyService>()
   private val omuService = mock<OmuService>()
   private val releaseDateService = mock<ReleaseDateService>()
+  private val probationSearchApiClient = mock<ProbationSearchApiClient>()
```

Comment:
No longer need these four in the LicenceService as we now have the LicenceCreation service and these are only used in licence creation.

Reply:
Done